client/src/components/CommentBody/CommentBody.js

```diff
@@ -0,0 +1,32 @@
+
+import React from 'react';
+import "../../App.css"
+import "./CommentBody.css";
+
+// Function based React Component
+const CommentBody = (props) => {
+  // Default Class to apply to Component
+  // No props right now but when there is data we can use 
+  // this.props.comment and this.props.date
+  let classList = `CommentBody`;
+  const body = props.bodyProps;
+  const date = props.dateProps;
+  const time = props.timeProps;
```

Comment:
Can you use destructing here to clean this up?

Also, you don't need to put `Prop` in the name of a prop (i.e. just `body`, not `bodyProp`)